docs/en/setup/backend/backend-win-monitoring.md

```diff
@@ -0,0 +1,34 @@
+# Windows Monitoring
+SkyWalking leverages Prometheus windows_exporter to collect metrics data from the Windows and leverages OpenTelemetry Collector to transfer the metrics to
+[OpenTelemetry receiver](opentelemetry-receiver.md) and into the [Meter System](./../../concepts-and-designs/meter.md).
+Windows entity as a `Service` in OAP and on the `Layer: OS_WINDOWS`.
+
+## Data flow
+**For OpenTelemetry receiver:**
+1. The Prometheus windows_exporter collects metrics data from the VMs.
+2. The OpenTelemetry Collector fetches metrics from windows_exporter via Prometheus Receiver and pushes metrics to the SkyWalking OAP Server via the OpenCensus gRPC Exporter or OpenTelemetry gRPC exporter.
+3. The SkyWalking OAP Server parses the expression with [MAL](../../concepts-and-designs/mal.md) to filter/calculate/aggregate and store the results.
+
+## Setup
+**For OpenTelemetry receiver:**
+1. Setup [Prometheus windows_exporter](https://github.com/prometheus-community/windows_exporter).
+2. Setup [OpenTelemetry Collector ](https://opentelemetry.io/docs/collector/). This is an example for OpenTelemetry Collector configuration [otel-collector-config.yaml](../../../../test/e2e-v2/cases/vm/prometheus-node-exporter/otel-collector-config.yaml).
```

Comment:
I think you should provide your own OTEL config.

Reply:
done.